src/Features/Core/Portable/Completion/Providers/AbstractObjectCreationCompletionProvider.cs

```diff
@@ -13,6 +13,7 @@
 using Roslyn.Utilities;
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using System;
+using Microsoft.CodeAnalysis.FindSymbols;
```

Comment:
sort usings.  Your sorting settings should be to place 'System' settings first.  Thanks